Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -23,6 +23,10 @@ public class PressableButton : MonoBehaviour, IMixedRealityTouchHandler
         [Tooltip("The object that is being pushed.")]
         private GameObject movingButtonVisuals = null;
 
+        [SerializeField]
+        [Tooltip("The icon and text content moving inside the button.")]
+        private GameObject movingButtonIconText = null;
```

Comment:
If you think we should put this in the general pressable button, then make sure to expose this so that developers can configure this at runtime.

Reply:
Sounds great!